extension/js/common/api/key-server/wkd.ts

```diff
@@ -67,7 +77,7 @@ export class Wkd extends Api {
       return { pubkey: null, pgpClient: null };
     }
     // if recipient uses same domain, we assume they use flowcrypt
-    const pgpClient = this.myOwnDomain === recipientDomain ? 'flowcrypt' : 'pgp-other';
+    const pgpClient = this.myOwnDomain === Str.getDomainFromEmailAddress(email) ? 'flowcrypt' : 'pgp-other';
```

Comment:
I suppose we'll be dropping pgpClient from everywhere. If you want to chunk up the upcoming PR, this could also be done in a separate PR.

Reply:
Let's drop it in the PR #3445 that actually "deletes" the `client` field from the database. This branch still attempts to store it.